### ISSUE TITLE
English hard coded in some javascript files (search.js for the cp)

### ISSUE THREAD

**Issue by robinsowell** [ExpressionEngine/ExpressionEngine#842]
There are a couple of spots in the control panel where there are no language keys because they're missing from javascript:

https://github.com/ExpressionEngine/ExpressionEngine/blob/ba9e8730b50a2fb7d9b78003bd4ac92b86302cf4/themes/ee/cp/js/src/vendor/codemirror/search.js#L74

https://github.com/ExpressionEngine/ExpressionEngine/blob/ba9e8730b50a2fb7d9b78003bd4ac92b86302cf4/themes/ee/cp/js/src/vendor/codemirror/search.js#L109

**Comment by intoeetive** [comment]
Since codemirror is third party script residing in vendor folder, I don't think we can do much here (unless we adopt version that supports localization - or work on creating one)

**Comment by robinsowell** [comment]
Ouch.  I don't love any of the options, including just closing it as unfixable.  It's a small thing, but also irksome.

https://codemirror.net/6/examples/translate/

I did not dig deep on that, but viable option?

**Comment by intoeetive** [comment]
So they are basically offering to create translations ourselves. In theory we can do that, though I would prefer an easier way :)
With other third-party tools (CKEditor and Redactor) we just include their language files.

**Comment by robinsowell** [comment]
Right- I think we could get away with just documenting it so if someone rolls a language pack, they could include this as well.  But it would need to swap out based on the EE language setting.  No clue how that would work.

And granted, it's not a great solution.  But I don't like it being static and stuck in English if there's a fairly reasonable alternative..

**Comment by intoeetive** [comment]
I tried implementint that giude, but it seems to work only with CodeMirror 6 - and I believe we have older version.

So we'd need to update CodeMirror to newer version first
